src/java.desktop/windows/native/libawt/windows/ShellFolder2.cpp

```diff
@@ -1056,7 +1056,8 @@ JNIEXPORT jintArray JNICALL Java_sun_awt_shell_Win32ShellFolder2_getIconBits
             bmi.bmiHeader.biCompression = BI_RGB;
             // Extract the color bitmap
             int nBits = iconSize * iconSize;
-            long colorBits[MAX_ICON_SIZE * MAX_ICON_SIZE];
+            long * colorBits;
```

Comment:
Pointer declarations aren't consistent in the file, the function parameter is declared as `JNIEnv* env`. However, in the majority of cases the asterisk is near the variable name as in `const char *pLibName`. In either case, there's only one space:
```cpp
long *maskBits;
```
or
```cpp
long* maskBits;
```